public/lesswrong-editor/src/llm-autocomplete.ts

```diff
@@ -0,0 +1,230 @@
+import Plugin from '@ckeditor/ckeditor5-core/src/plugin';
+// import { ButtonView } from '@ckeditor/ckeditor5-ui';
+import TurndownService from 'turndown';
+
+
+export default class AIAutocomplete extends Plugin {
+    static get pluginName() {
+        return 'AIAutocomplete';
+    }
+
+    init() {
+        const editor = this.editor;
+
+        // editor.ui.componentFactory.add('autocompleteButton', () => {
+        //     const button = new ButtonView();
+
+        //     button.set({
+        //         label: 'AI Autocomplete',
+        //         tooltip: true,
+        //         withText: true
+        //     });
+
+        //     button.on('execute', () => {
+        //         this.autocomplete();
+        //     });
+
+        //     return button;
+        // });
```

Comment:
cleanup?  or do you think you might come back to this later

Reply:
Might come back to it later for user-rollout